ui/static/ui/css/mit-lore.css

```diff
@@ -622,3 +622,35 @@ p.confirmation-description {
     font-style: normal;
     font-weight: normal;
 }
+
+i.add-vocabulary-spinner-hide {
```

Comment:
We don't need to use CSS here to hide and show the spinner. Please change the classes to `add-vocabulary-spinner` and `lr-spinner` and remove the `display` style attributes.